packages/react-vapor/rollup.config.js

```diff
@@ -47,69 +35,82 @@ export default [
         'redux',
         'underscore',
         'coveo-styleguide',
-        'underscore.string',
+        'underscore.string'
     ],
     onwarn,
     plugins: [
         inject({
-            jQuery: 'jquery', // chosen-js expects jQuery to be available as a global
+            jQuery: 'jquery' // chosen-js expects jQuery to be available as a global
         }),
         replacePlugin(),
         postcss({
             extract: false,
             modules: {localIdentName: '[name]-[local]-[hash:base64]'},
             namedExports: true,
-            use: ['sass'],
+            use: ['sass']
         }),
         scssVariable(),
         resolve({
-            browser: true,
+            browser: true
         }),
         commonjs({
             namedExports: {
                 'hogan.js': ['Template', 'compile'],
                 'react-modal': ['setAppElement'],
                 'react-dnd': ['DragDropContext', 'DropTarget', 'DragSource'],
-                diff2html: ['diffChars', 'diffWordsWithSpace'],
-            },
+                diff2html: ['diffChars', 'diffWordsWithSpace']
+            }
         }),
         tsPlugin(),
-        ...(file.includes('.min.') ? [terser()] : []),
-    ],
+        externalGlobals({
+            codemirror: 'CodeMirror',
+            d3: 'd3',
+            jquery: '$',
+            react: 'React',
+            'react-dom': 'ReactDOM',
+            'react-redux': 'ReactRedux',
+            redux: 'Redux',
+            underscore: '_',
+            'coveo-styleguide': 'VaporSVG',
+            'underscore.string': 's',
+            'react-dom/server': 'ReactDOMServer'
+        }),
+        ...(file.includes('.min.') ? [terser()] : [])
+    ]
 }));
 
 function tsPlugin() {
     return typescript({
         transformers: [
-            (service) => ({
+            service => ({
```

Comment:
it seems like prettier hasn't used our settings, normally it should enforce having parentheses 🤔

Reply:
The linter did that when I commited... :thinking:

Reply:
but when I saved prettier did use our config